webapp/packages/plugin-connection-custom/src/DriverSelector/Driver.module.css

```diff
@@ -5,8 +5,24 @@
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
+.icon {
+  position: relative;
+}
+
 .staticImage {
   box-sizing: border-box;
   width: 24px;
   max-height: 24px;
-};
+}
+
+.indicator {
+  position: absolute;
+  bottom: 2px;
+  right: 0;
+  width: 12px;
+  height: 12px;
```

Comment:
may be we need a bit more larger icon? this one seems too small for me
![image](https://github.com/user-attachments/assets/5702044c-9403-4f59-93f5-e579e0d17c78)

Reply:
Yep, If we expect users to hover over that icon, it's better to have it at least 14px or 16px. [WCAG recommends minimum 24px](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html), however in our case it's not a button and maybe not so critical, so we can make it smaller.
And I personally would place it on top. For users it will make more sense in terms of catching their attention since such approach is commonly used among many applications for different type of notifications.

Reply:
Actually, I have a better Idea. Why don't we just make the whole driver icon hoverable? I assume this is how it should work. Icon itself doesn't need to be a target